crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -14,6 +14,8 @@ const DPKI_NETWORK_SEED_TESTING: &str = "deepkey-testing";
 
 /// Configure which app instance ID to treat as the DPKI application handler
 /// as well as what parameters to pass it on its initialization.
+/// Note that these settings have an effect on network compatibility,
+/// see [`holochain_p2p::spawn::NetworkCompatParams`].
```

Comment:
It's only the DNA hash that affects network compatibility which is a combination of the loaded DNA and the `network_seed` field right? Can we make that explicit in these docs?

Reply:
Nope, the DPKI "id" affects network compat too, via `NetworkCompatParams`

Reply:
I must say that the description
``` Configure which app instance ID to treat as the DPKI application handler
/// as well as what parameters to pass it on its initialization.
```
is cryptic to me and I don't understand it. What's the app instance ID/DPKI id and where is it set in this struct?